### ISSUE TITLE
✨ Experiment - SEO Table

### ISSUE THREAD

**Issue by Harry-Ross** [SSWConsulting/SSW.Website#1380]
Timebox - 2hrs

cc @camillars @sethdailyssw 

As per my conversation with @bradystroud, it would be great to create a page that can display the SEO descriptions and titles for each page, and allow editing of them. 

```[tasklist]
### Tasks
- [x] Create a proof of concept page where the SEO on pages can be edited using https://tina.io/docs/graphql/queries/update-document/
- [x] Determine if this page is feasible and worth building
```

**Comment by sethdaily** [comment]
+1 good idea

**Comment by Harry-Ross** [comment]
As per my conversation with @bradystroud @wicksipedia, I have found that at the moment it is not feasible to use the GraphQL API to write to TinaCMS. The work required to access the GitHub API to modify the markdown files would be a lot of effort (couple days at least), and likely shouldn't be prioritised compared to the migration of other pages. 

I have created a read-only page for the SEO values of every page, but this is likely not useful.
